sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
I also addressed the problem in this comment: https://github.com/scikit-learn/scikit-learn/pull/5204#issuecomment-137265512

There are three ways I think we can get around this
* the current block of code to which this discussion is attached
* by forcing the solver to be `lbfgs` in `test_probability` (as I did in 9d3becf)
* do neither, but change that "degenerate" case at line ~290 in `test_probability`